app/connectors/SubmittedProcessConnector.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.AppConfig
+import connectors.httpParsers.SaveSubmittedProcessHttpParser.saveSubmittedProcessHttpReads
+import javax.inject.{Inject, Singleton}
+import models.{RequestOutcome, SaveSubmittedProcessResponse}
+import play.api.libs.json.JsValue
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class SubmittedProcessConnector @Inject()(httpClient: HttpClient, appConfig: AppConfig) {
+
+  def makeAvailableForApproval(process: JsValue)
+                              (implicit ec: ExecutionContext, hc: HeaderCarrier): Future[RequestOutcome[SaveSubmittedProcessResponse]] = {
+
+    val endpoint: String = appConfig.externalGuidanceBaseUrl + "/external-guidance/submitted"
```

Comment:
Is this end point correct? Looking at EG-255 for External guidance I think the final path element is /submit.

Reply:
in EG-255 end point is /submitted

Reply:
I see the branch as been updated.